Dockerfile.client

```
@@ -7,7 +7,9 @@ WORKDIR '/app'
 
 COPY package.json package-lock.json ./
 USER node
-RUN npm ci
+
+RUN rm -rf node_modules && npm cache clean --force && npm install --production
```

I think olny run _npm install --production_ shoud do it.
Why: ist starts with an folder who only inclued to files _package.json_ and _package-lock.json_. There is no folder like node_modules or even something at the npm cahce.
```suggestion
RUN npm install --production
```

---

doing npm install will ignore the package-lock.json leading to non reproduce able builds - don't do that!